hypervisor/include/arch/x86/vtd.h

```diff
@@ -530,8 +530,10 @@ int unassign_iommu_device(const struct iommu_domain *domain, uint8_t bus, uint8_
  * @param[in] translation_table the physcial address of EPT table of the VM specified by the vm_id
  * @param[in] addr_width address width of the VM
  *
- * @return Pointer pointer to iommu_domain
- * @return NULL if translation_table is 0
+ * @return Pointer to the created iommu_domain
+ *
+ * @retval NULL when \p translation_table is 0
+ * @retval Non-NULL when \p translation_table is not 0
```

Comment:
Could we use `<0` here instead of `Non-NULL` (assuming the return value is always negative)?

(the automatic formatting will make whatever follows `retval` formatted like a parameter, or code so this is why I prefer to avoid plain English here)

Reply:
`<0` may be confusing here because the return  value is a pointer, not an integer.